cmd/plugin/krew/krew.go

```diff
@@ -99,11 +99,11 @@ func RootCmd(ctx context.Context, o *krewOptions) *cobra.Command {
 
 	cmd.RunE = func(cmd *cobra.Command, args []string) error {
 		if len(args) != 1 {
-			return ErrPodRequired
+			o.pod = ""
```

Comment:
do you need to set it empty string? don't see where it could be set to something non-empty since initialization